ci/datamgr.py

```diff
@@ -416,6 +416,36 @@ def omniscidb(schema, tables, data_directory, **params):
     conn.close()
 
 
+@cli.command()
+@click.option('-h', '--host', default='localhost')
+@click.option('-P', '--port', default=1433, type=int)
+@click.option('-u', '--user', default='SA')
+@click.option('-p', '--password', default='Ibis_MSSQL_2017')
+@click.option('-D', '--database', default='master')
+@click.option(
+    '-S',
+    '--schema',
+    type=click.File('rt'),
+    default=str(SCRIPT_DIR / 'schema' / 'mssql.sql'),
+)
+@click.option('-t', '--tables', multiple=True, default=TEST_TABLES)
+@click.option('-d', '--data-directory', default=DATA_DIR)
+def mssql(schema, tables, data_directory, **params):
```

Comment:
I am not very famliar with database set ups..Can you please explain how is this method used?

Reply:
This method is used to initalized the test dataset for the CI for sql server